server/src/app.middlewares.js

```diff
@@ -113,11 +113,12 @@ export function setAppMiddlewares(app, doneLoading) {
         by doing this, route code is worry-free and no need to pass "req{}" to gdc purpose-specific code doing the API calls
         these *protected* contents are not used in non-gdc code
         */
-		req.query.__protected__ = {}
+		const __protected__ = { ignoredTermIds: [] }
```

Comment:
Can we rename __protected__ to __userData__ or __protectedUserData__ or something? Something that indicates what this variable stores